### ISSUE TITLE
feature request: Visualization in IDE with the plugin

### ISSUE THREAD

**Issue by razakpm** [kumarharsh/graphql-for-vscode#28]
Provide an option to view GraphQL visualization in IDE itself.

**Comment by kumarharsh** [comment]
@razakpm do you mean this: http://nathanrandal.com/graphql-visualizer/

**Comment by razakpm** [comment]
@kumarharsh , Yes we also using https://apis.guru/graphql-voyager/ or https://github.com/sheerun/graphqlviz
